packages/core/src/hostconfiguration.ts

```diff
@@ -42,4 +43,9 @@ export interface HostConfiguration {
    * A map of secret name and their respective regex pattern
    */
   secretPatterns?: Record<string, string>;
+
+  /**
+   * MCP server configuration. The tools will be injected into the prompt.
```

Comment:
this map is used as a lookup table for MCP servers. Script still need to include the MCP server id and it will be filled with the information from this configuration.